guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,375 @@
+The coin(s) selected for a transaction determine the derived on-chain address due to the nature of silent payments protocol. The interface should avoid showing the on-chain address before the coin selection is done.
+
+{% include tip/close.html %}
+
+
+### Test transactions
+
+Test transactions are another way to alleviate user anxiety for sending on-chain payments, especially with static addresses, since the on-chain address is always visibly different from the static address.
+
+{% include image-gallery.html pages = page.images_test-txns %}
+
+## Receiving & scanning
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/static-address-with-customization-options.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/static-address-with-customization-options@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/static-address-with-customization-options.png"
+   width = 250
+   height = 522
+   alt-text = "An application screen showing labels being added to a static address."
+   caption = "."
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Since static addresses can now be used to pay a receiver several times, receiving flows are likely to be used less often, rarely even, since static addresses can be posted in public and others can simply get it from there. So whenever the receiver is sharing a payment request, users have the opportunity to associate valuable information to it – through labels and contacts; applications should facilitate this process.
+
+Some places where people can share static addresses (and add labels) to include:
+- Social media pages
+- Donate or fundraising websites
+- Exchanges
+- Email signatures
+
+</div>
+
+
+Receivers using mobile wallets, which may not be online 24x7 will encounter some delay (once they come online) as their wallet scans the blockchain and performs calculations to detect their payments. Applications should communicate this fact during setup/onboarding. While the scanning itself is taking place, applications should show progress data and estimated time to complete the scanning process.
+
+The time to detect a specific received payment can be minimized if the sender & receiver can communicate with each other. Some applications already allow users to detect whether an address belongs to them. This feature should, if possible, be extended to include BIP-352 functionality.
+
+## Backup
+
+Like lightning wallets, on-chain bitcoin wallets supporting full-featured silent payments require file backups (in the cloud or offline) since the wallet has important [user metadata](/guide/daily-spending-wallet/backup-and-recovery/landing-page/#user-metadata) such as contacts and labels, besides the private key (recovery phrase). These backup files may also contain:
+- recovery phrase (for BIP 39 wallets)
+- Scan and/or spend keys
+- labels & even contacts
+- transaction id's
+- wallet birthday
+
```

Comment:
```
- recovery phrase (for BIP 39 wallets)
- Scan and/or spend keys
- labels & even contacts
- transaction id's
- wallet birthday
```

Just noticed that only the second bullet point started with a capital letter.

Reply:
Fixed in https://github.com/BitcoinDesign/Guide/pull/1109/commits/3e040a510d5a560e6947e9e85ee7446bad7fdad8